File: src/main/java/ar/edu/um/haberes/rest/model/Contacto.java
/**
 * 
 */
package ar.edu.um.haberes.rest.model;

import java.io.Serializable;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;

import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

/**
 * @author daniel
 *
 */
@Entity
@EqualsAndHashCode(callSuper = false)
@Getter
@Setter
@ToString
@NoArgsConstructor
@AllArgsConstructor
public class Contacto extends Auditable implements Serializable {
	/**
	 * 
	 */
	private static final long serialVersionUID = -6142798960269045664L;

	@Id
	private Long legajoId;

	private String fijo = "";
	private String movil = "";
	private String mailPersonal = "";
	private String mailInstitucional = "";

}

File: src/main/java/um/haberes/rest/kotlin/model/Contacto.kt
package um.haberes.rest.kotlin.model

import um.haberes.rest.model.Auditable
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table
data class Contacto(

    @Id
    var legajoId: Long? = null,
    var fijo: String = "",
    var movil: String = "",
    var mailPersonal: String = "",
    var mailInstitucional: String = ""

) : Auditable()
